src/test/resources/org/prebid/server/it/openrtb2/rubicon_appnexus/test-rubicon-bid-request-2.json

```diff
@@ -126,7 +126,6 @@
     "ext": {
       "schain": {
         "ver": "1.0",
-        "complete": 1,
```

Comment:
This parameter is returned there, after you fix parameter name from cmplete -> complete